core/src/main/scala/org/apache/spark/sql/delta/commands/cdc/CDCReader.scala

```diff
@@ -154,6 +157,16 @@ object CDCReader extends CDCReaderImpl
       timestamp: Timestamp,
       actions: Seq[T],
       commitInfo: Option[CommitInfo])
+
+  case class FileVersion(version: Long, timestamp: Timestamp) {
```

Comment:
One or two lines doc describing what this is used for. Same comment for below case class.

Reply:
Done!